libs/remix-debug/src/debugger/debugger.ts

```diff
@@ -41,15 +41,19 @@ export class Debugger {
       }
     })
 
-    this.breakPointManager.event.register('managersChanged', () => {
-      const { traceManager, callTree, solidityProxy } = this.debugger
-      this.breakPointManager.setManagers({ traceManager, callTree, solidityProxy })
-    })
-
     this.breakPointManager.event.register('breakpointStep', (step) => {
+      this.event.trigger('breakpointStep', [])
```

Comment:
Is this a recursive call? Because i don't see anywhere else that `breakpointStep` event is registered. If not, then i'm not sure how useful this line is.